### ISSUE TITLE
`migration --run-syncdb` doesn't create models

### ISSUE THREAD

**Issue by graingert** [zenwalker/django-micro#14]
https://github.com/django/django/blob/a77f21880dfb0631ea0adb22d47909915cbfc3a9/django/core/management/commands/migrate.py#L268-L276

```python
        # Build the manifest of apps and models that are to be synchronized.
        all_models = [
            (
                app_config.label,
                router.get_migratable_models(app_config, connection.alias, include_auto_created=False),
            )
            for app_config in apps.get_app_configs()
            if app_config.models_module is not None and app_config.label in app_labels
        ]
```

It seems that `app_config.models_module` is None for django-micro apps

## Reproduction:

| Problem | Repro |
|------------|----------|
| only works with app_name and migrate | https://pyfiddle.io/fiddle/f5a6361a-49df-4c79-8d3c-f94cadb94b6a |
|makemigrations needs app_name | https://pyfiddle.io/fiddle/ca85bac0-b44b-4810-9a4f-4afd9f6440a4
|migrate sync_db=True doesn't work | https://pyfiddle.io/fiddle/2512299d-6fdb-4e36-9612-96844d4122f9 |

**Comment by graingert** [comment]
reproduction: https://pyfiddle.io/fiddle/af018023-317c-4460-9d76-9e40621f3def/?i=true

turns out [pyfiddle doesn't support sqlite3](https://github.com/priyankcommits/pyfiddleio/issues/3) so you'll have to copy it into your codebase

**Comment by maxpoletaev** [comment]
Thank you for reporting.  I’ll look into it.

**Comment by graingert** [comment]
```python
import django_micro
from django.http import HttpResponse
from django.db import models

django_micro.configure({
    'DEBUG': True,
    'DATABASES': {
        'default': {
            'ENGINE': 'django.db.backends.sqlite3',
            'NAME': ':memory:'
        },
    },
})

@django_micro.route('', name='homepage')
def homepage(request):
    name = request.GET.get('name', 'World')
    return HttpResponse('Hello, {}!'.format(name))


class Post(models.Model):
    title = models.CharField(max_length=255)
    parent = models.ForeignKey('self', null=True, on_delete=models.CASCADE)

    class Meta:
        app_label = django_micro.get_app_label()


from django.core import management

@django_micro.command('print_hello')
class PrintHelloCommand(management.BaseCommand):
    def handle(self, *args, **options):
        management.call_command('makemigrations', django_micro.get_app_label())
        management.call_command('migrate', run_syncdb=True)
        p = Post.objects.create(title='foo')
        Post.objects.create(parent=p, title='ham')
        print(
            Post
            .objects
            .values_list('parent__title', 'title')
        )


application = django_micro.run()
```

**Comment by graingert** [comment]
also interestingly omitting the `django_micro.get_app_label()` in the `makemigrations` command means it django doesn't find any changes

**Comment by graingert** [comment]
@zenwalker okay I've managed to get sqlite working on pyfiddle: https://pyfiddle.io/fiddle/f5a6361a-49df-4c79-8d3c-f94cadb94b6a/

**Comment by maxpoletaev** [comment]
Fixed in 1.7.1 🎉
